pkg/controller/pod_volume_backup_controller.go

```diff
@@ -213,6 +219,28 @@ func (c *podVolumeBackupController) processBackup(req *arkv1api.PodVolumeBackup)
 		req.Spec.Tags,
 	)
 
+	// if this is azure, set resticCmd.Cmd().Env to be os.Environ() plus azure.GetResticEnvVars
```

Comment:
How would you feel about extracting this to it's own method? I think this method is already just a procedural list of things to do and check for errors, and having the specific azure logic here makes the natural flow or reading this feel interrupted.